src/libs/algolia/fetchAlgolia/transformAlgoliaHit.ts

```diff
@@ -41,6 +41,10 @@ const parseGeoloc = (hit: AlgoliaHit): AlgoliaHit['_geoloc'] =>
 export const filterAlgoliaHit = (hit: IncompleteSearchHit): boolean =>
   hit && hit.offer && !!hit.offer.thumbUrl && typeof hit.offer.subcategoryId !== 'undefined'
 
+// We don't want to display venues without image
+export const filterVenueHit = (hit: IncompleteVenueHit): boolean =>
+  hit && typeof hit.bannerUrl === 'string'
```

Comment:
The type cast `as VenueHit` should be in the filter

Reply:
comment ça ?